core/templates/dev/head/pages/exploration_editor/history_tab/CompareVersionsService.js

```diff
@@ -82,6 +83,7 @@ oppia.factory('CompareVersionsService', [
        * Should be called after VersionTreeService.init() is called.
        * Should satisfy v1 < v2.
        */
+      
```

Comment:
No need to leave this line here. We do leave a line after the @fileoverview comment because it gives the overview of the entire file. Generally no space is left between the function definition and the comment documenting it.